x-pack/legacy/plugins/spaces/server/new_platform/spaces_service/spaces_service.ts

```diff
@@ -66,6 +69,41 @@ export class SpacesService {
       return spaceId;
     };
 
+    const getScopedClient = async (request: RequestFacade) => {
```

Comment:
No functional changes; just moved from its original location below so it could be reused by `getActiveSpace`